frontend/integration-tests/tests/kubevirt/vm.actions.scenario.ts

```diff
@@ -34,23 +35,34 @@ describe('Test VM actions', () => {
 
     it('Starts VM', async() => {
       await listViewAction(vmName)('Start');
-      await browser.wait(until.textToBePresentInElement(listViewVMmStatus(vmName), 'Running'), VM_BOOTUP_TIMEOUT);
+      await browser.wait(
+        until.and(
+          until.presenceOf(listViewVmIcon(vmName, runningIcon)),
+          until.textToBePresentInElement(listViewVmStatus(vmName), 'Running')
+        ), VM_BOOTUP_TIMEOUT);
     });
 
     it('Restarts VM', async() => {
       await listViewAction(vmName)('Restart');
-      await browser.wait(until.textToBePresentInElement(listViewVMmStatus(vmName), 'Starting'), VM_BOOTUP_TIMEOUT);
-      await browser.wait(until.textToBePresentInElement(listViewVMmStatus(vmName), 'Running'), VM_BOOTUP_TIMEOUT);
+      await browser.wait(
+        until.and(
+          until.presenceOf(listViewVmIcon(vmName, pendingIcon)),
+          until.textToBePresentInElement(listViewVmStatus(vmName), 'Starting')
+        ), VM_BOOTUP_TIMEOUT);
+      await browser.wait(until.and(
+        until.presenceOf(listViewVmIcon(vmName, runningIcon)),
+        until.textToBePresentInElement(listViewVmStatus(vmName), 'Running'))
+        , VM_BOOTUP_TIMEOUT);
     }, VM_ACTIONS_TIMEOUT);
 
     it('Stops VM', async() => {
       await listViewAction(vmName)('Stop');
-      await browser.wait(until.textToBePresentInElement(listViewVMmStatus(vmName), 'Off'), VM_STOP_TIMEOUT);
+      await browser.wait(until.presenceOf(listViewVmIcon(vmName, offIcon)), VM_STOP_TIMEOUT);
```

Comment:
should we add id to VmStatus component to easier locate the text? Maybe there should be more ids in some other components?

Reply:
I have discussed this with Vojtech, the thing is that the status text is not a link when the VM is off, so we would probably need to add it into a span. It would be definitely easier if it had an ID. But I'm considering removing the text checks altogether and only wait for the specific icon to stay independent on localization. Suggestions?

Generally, VM overview is missing ids, for example Flavor, template, memory, name, description would all be nice to have marked with ids or classes.

Reply:
ok,
I added bunch of ids to https://github.com/kubevirt/web-ui-components/pull/248. Please look at the PR if some other ids are needed.

if you by any chance need the id of statuses, ping this PR: https://github.com/kubevirt/web-ui-components/pull/216

Reply:
@suomiy  Looks good